doc/manual/src/contributing/hacking.md

```diff
@@ -1,5 +1,8 @@
 # Hacking
 
+## Flakes
+This section assumes you are using flakes-enabled nix. See the next section for equivalent instructions without the use of flakes.
```

Comment:
Yeah, the intention is not to break *everything* into one-sentence paragraphs. That one appears to be a fluke due to muscle-memory, because people tend to write way too long paragraphs.

Revised the suggestion.

[Short paragraphs are recommended by the plain language guidelines](https://www.plainlanguage.gov/guidelines/concise/write-short-paragraphs/) developed by the US government and supported by evidence that it indeed does help understanding.